lib/dialects/mysql/query-generator.js

```diff
@@ -158,6 +159,100 @@ const QueryGenerator = {
     return `ALTER TABLE ${this.quoteTable(tableName)} CHANGE ${attrString.join(', ')};`;
   },
 
+  handleSequelizeMethod(smth, tableName, factory, options, prepend) {
```

Comment:
Looks same as SQLite's, is there any difference? SQLite inherits MySQL dialect, I would like to minimize duplicated code

Reply:
They're different, and they both call `AbstractQuery.handleSequelizeMethod()` at the end, so there's no duplicated code here. I did however remove the `_traverseJSON` method which was duplicated.